lib/quickbooks/model/base_model.rb

```diff
@@ -12,6 +12,48 @@ def initialize(attributes={})
         attributes.each {|key, value| public_send("#{key}=", value) }
       end
 
+      def self.all query = nil, options = {}
```

Comment:
`query_in_batches()` will continue working fine unless user set `allow_pagination` to false. If `allow_pagination` is set to false then no pagination will happen.